utils/hare-node

```diff
@@ -17,9 +17,39 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
-set -eu -o pipefail
-# set -x
-export PS4='+ [${BASH_SOURCE[0]##*/}:${LINENO}${FUNCNAME[0]:+:${FUNCNAME[0]}}] '
+# :help: stop the cluster
 
-# :help: manage the cluster nodes
-PCSCLI_PROG_NAME='hctl node' pcswrap "$@"
```

Comment:
@gshipra, we need to keep this code, I think we must add a subcommand to `hctl node` as `hctl node pcs`?
what do you think @knekrasov? We need to inform CSM team to change the command invocation in that case.

Reply:
older script was in shell. This new code is in python. How to add this functionality in python ?